.travis.yml

```diff
@@ -15,21 +15,13 @@ jobs:
         - make test
         - pip install safety
         - safety check || true
-    - name: “Python 3.8 make lint and selected pytests”
-      python: "3.8"
-      before_script: true  # override npm install below
-      script:
-        # On pull requests, run all flake8 tests on modified code
-        # if clause avoids `fatal: ambiguous argument 'origin/master'` on git push
-        - if [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
-            make lint-diff;
-          fi
-        - source scripts/run_doctests.sh
-        - source scripts/test_py3.sh
     - name: “Python 3.8 on focal”
       python: "3.8"
+    - name: “Python 3.9-dev on focal”
+      python: "3.9-dev"
   allow_failures:
     - name: “Python 2.7 on Infogami master”
+    - python: "3.9-dev"
```

Comment:
We want this to be on 2.7+master, don't we?

Reply:
[The output](https://travis-ci.org/github/internetarchive/openlibrary/builds/721516307) shows that two runs are currently allowed to fail.  All runs with the `name: “Python 2.7 on Infogami master”` and also all runs with `python: "3.9-dev"`.  I believe that this is what we want.

Reply:
🤦 Gaaah, I completely misread this! Sorry, that makes complete sense :P